### DISCUSSION TITLE
Add information to README.md

### DISCUSSION THREAD

**Discussion by fondesa** [androiddevelopersitalia/aditogether#59] [General]
Currently the README file looks a bit empty and also outdated on the setup steps.
On the other hand, `CONTRIBUTING.md` written by @mcatta contains all the necessary information to contribute to the project.

We should add at least the following sections:
- short description of the project
- `Contributing to the project` or something like that with a link to our `CONTRIBUTING.md`.

Also, maybe we should remove the empty sections and add them when they are ready (e.g. tech stack)?

What do you think?

Have you some proposals to make the README look better in the short term (e.g. like `CONTRIBUTING.md`)?

@4face-studi0 @mcatta

**Comment by fardavide** [comment]
Good points!
I think we could start populate the teck stack tho, as we already decided to use Groovy and Kotest (`BehaviorSpec`)

**Comment by mcatta** [comment]
I agree, we could add a small section about the structure of the project, as well. Something like:
```
build-tools
|- common
|- jvm
|- android 
FeatureA
|- ui
|- data
|- domain
...
```
